contracts/AutomationBot.sol

```diff
@@ -277,6 +289,7 @@ contract AutomationBot {
     ) external onlyDelegate {
         require(triggerData.length > 0, "bot/remove-at-least-one");
         address automationBot = serviceRegistry.getRegisteredService(AUTOMATION_BOT_KEY);
+        AutomationBot(automationBot).clearlock();
```

Comment:
is it required here ? Or "just in case" ? :)
there is no check for lockCount in `removeTriggers` or `removeRecord`

Reply:
for consistency, at the start of execution lock is expected to be false